doc/changelog/03-notations/13519-primitiveArrayNotations.rst

```diff
@@ -0,0 +1,9 @@
+- **Added:**
+  :cmd:`Number Notation` and :cmd:`String Notation` can now register parsers
+  and printers for terms containing primitive floats, primitive arrays
+  and type constants for primitive types.
```

Comment:
```suggestion
  :cmd:`Number Notation` and :cmd:`String Notation` now support
  parsing and printing of primitive floats, primitive arrays
  and type constants of primitive types.
```

Simpler and clearer--does it capture the essence of the change?  `type constants` or can it just be `constants`?

Reply:
It should be type constants, as `constants` sounds too much line ints themself instead of their type imho